x/evm/keeper/grpc_query.go

```diff
@@ -254,8 +256,17 @@ func (k Keeper) EthCall(c context.Context, req *types.EthCallRequest) (*types.Ms
 	return res, nil
 }
 
+const (
+	FromTypeRPC      = 1
+	FromTypeInternal = 2
+)
```

Comment:
define as type and please add comments to explain each of these
```suggestion
type CallType int

const (
	RPC      CallType = iota + 1
	Internal
)
```